app/i18n/locales/en-US.json

```diff
@@ -66,6 +66,7 @@
   "daedalusTransfer.waiting.progressInfo.checkingAddresses": "Checking addresses funds",
   "daedalusTransfer.waiting.progressInfo.restoringAddresses": "Fetching addresses",
   "daedalusTransfer.waiting.title.label": "Daedalus wallet is being restored",
+  "daedalusTransfer.waiting.warning": "⚠️ This may take a long time or fail on poor internet connections",
```

Comment:
not sure how crowdin react on ⚠️  char :)

Reply:
I don't see why it would be a problem given it supports the rest of unicode just fine